### DISCUSSION TITLE
Help with website launch.

### DISCUSSION THREAD

**Discussion by hansemso** [community/community#43724] [New to GitHub] [answered]
I am stumped.  I got my website up.  And then it disappeared.  It says "Your site is live at https://hansemso.github.it.  But when I lick the link get the message:  404 File Not Found.  Do I have to upgrade, pay the $48 to solve this problem or something?  Help.

**Comment by cx0y** [comment]
Hello @hansemso ! If you're deploying from a private repo then you have to upgrade your GitHub subscription

**Comment by melvincwng** [comment]
Hey @hansemso, welcome to GitHub 👋 

Understand you may have some issues setting up your very first website on GitHub Pages.

I took a look at your [repository](https://github.com/hansemso/hansemso.github.io), and based on your commit history, I would suggest you to do two things:

1. Create a index.html file (don't use other names for the html file)
2. Ensure that you are publishing your GitHub page from the correct source in the settings.

Some useful references for you: 
1. [GitHub Pages - How to set up a github.io page](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site)
2. [GitHub Pages - How to publish from the correct source/branch](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site)

Hope this solves your issue 😃

**Reply by melvincwng** [reply]
@hansemso, you need to create a index.html file in order for your GitHub Pages to work. However, based on your commit history, I see the following:

![image](https://user-images.githubusercontent.com/77479885/211335427-6a5a8cf3-e78f-4d0e-81ed-dfc4aa05a1ff.png)

If you rename the file to 'index.html' & then select the correct source/branch to publish the page, it should resolve this issue.

Hope this clarifies & answers your question :smile:

**Reply by cx0y** [reply]
Oh! This is the problem and I thought https://hansemso.github.it is a custom domain. 😄

**Reply by melvincwng** [reply]
@cx0y, I think that could be a typo. At least that's what I deduced from the author's commit history in his/her repository.

**Reply by hansemso** [reply]
Thanks.  I did that.  I made a index.html file using Visual Code.  I put in it a basic "hello world" code I got online, saved it as html, uploaded.  Now when I click "Visit site" all I see is a white blank page.  I guess that's better than the 404 file not found message.  But "Hello world" not showing.

**Reply by melvincwng** [reply]
@hansemso, based on your commit over [here](https://github.com/hansemso/hansemso.github.io/commit/fbba8f2c3eb9baeae271aee5799d536c5ff71bbd), I can see that your HTML code doesn't have any content in the <body> tags.

Hence, that's why your github.io page is showing as a white blank page.

You need to insert some markup or content into the <body> tags for it to appear in your github.io page.

Hope this clarifies & answers your question.

**Reply by melvincwng** [reply]
@hansemso, btw I verified on my end & your website is indeed working now.

You just need to insert some content as I've mentioned earlier.

Enjoy your new website :star:

**Reply by cx0y** [reply]
![Screenshot_20230109_204431](https://user-images.githubusercontent.com/93507446/211341578-aa2a6d06-1b67-418d-bef8-3f9aba7a1ff1.png)

there is no `hello world`

**Reply by melvincwng** [reply]
@hansemso, your website has been successfully deployed.

For your reference:
![image](https://user-images.githubusercontent.com/77479885/211343017-8aa1f1ac-beb8-45f5-9f13-7e05a70c7876.png)

If this answer has been helpful, please kindly 'Mark as answer' so that others know this issue has been resolved. 

Thanks, and have a nice day ahead :smile:

**Reply by hansemso** [reply]
oops.  Somehow the words got erased when I copied and pasted off the internet.  Now I see the words.  Thanks.  It's a start!

**Comment by hansemso** [comment, marked as answer]
How do i do that?  I don't see a button anywhere.  But my site is down again.  I put in my final code as index.html.  I copied the code for a template for a personal website and it doesn't show.

**Reply by cx0y** [reply]
But https://hansemso.github.io seems ok to me!

**Reply by melvincwng** [reply]
@hansemso I can also confirm your website looks ok as well with the boilerplate template.

**Reply by hansemso** [reply]
Oh, yeah.  One template didn't work, so I tried another one and it works fine.
